### PR TITLE
Update ARIA spec references to resolve ReSpec warnings

### PR DESCRIPTION
**Pull request by Malvoz**
`Bad reference: [ARIA] (appears 1 times)`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -306,7 +306,7 @@ <h3>The Maps for HTML Community Group</h3>
 The initial requirements sketched out at LGD2014
 were expanded into <a href="draft-2015.html">an initial use cases and requirements summary</a>.
 That work was the basis of the draft [[[MapML]]] specification
-and [[[HTML-map-element]]] proposal.
```
Comment (Malvoz) [inline index.html:309]: This currently says:

> and [_The HTML map Element proposal_](https://maps4html.org/HTML-Map-Element/spec/) proposal.

File: index.html [hunk 2]
```diff
@@ -4399,7 +4399,7 @@ <h5>Conclusion</h5>
 <li><a data-ucr-role="tag" href="">accessibility-potential</a>
 A native implementation would be better able to incorporate a semantically meaningful representation of a map
 into its accessibility tree than current implementations,
-which depend on the accessibility features of more general native elements in conjunction with [[ARIA]].
+which depend on the accessibility features of more general native elements in conjunction with [[WAI-ARIA]].
```
Comment (AmeliaBR) [inline index.html:4402]:
```suggestion
which depend on the accessibility features of more general native elements in conjunction with [[[wai-aria-1.1]]].
```

Without the number it links to version 1, and 1.1 is already fully stable (W3C Recommendation).

File: index.html [hunk 3]
```diff
@@ -5425,7 +5425,7 @@ <h5>Conclusion</h5>
 </li>
 <li><a data-ucr-role="tag" href="">accessibility-potential</a>
 Current implementations can only represent changes in the map state in a semantically meaningful form
-to a limited degree via such mechanisms as [[[ARIA]]] live regions.
+to a limited degree via such mechanisms as [[WAI-ARIA]] <a href="https://www.w3.org/TR/wai-aria/#dfn-live-region">live regions</a>.
```
Comment (AmeliaBR) [inline index.html:5428]:
```suggestion
to a limited degree via such mechanisms as [[[wai-aria-1.1]]] <a href="https://www.w3.org/TR/wai-aria/#dfn-live-region">live regions</a>.
```

(The W3 URL without the version, in contrast, always links to the latest stable version! But SpecRef doesn't seem to have a way to do that.)

**Review by AmeliaBR — CHANGES_REQUESTED** [review]
Thanks for submitting this. I spotted that ugly red wiggly error line yesterday, but I had other things to focus on.

Minor fix related to SpecRef versioning, I'll see if I can commit the suggested changes myself & then merge.

File: index.html (hunk 2 above)
Comment (Malvoz) [reply index.html]: That's only the attribution:

> _Accessible Rich Internet Applications (WAI-ARIA) 1.0. James Craig; Michael Cooper et al. W3C. 20 March 2014. W3C Recommendation. URL: https://www.w3.org/TR/wai-aria/_

however the link would still take you to the latest recommendation. But I accept your changes.

**Pull request merged**